src/index.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable */
+import _AXACheckbox from './components/a-checkbox';
```

Comment:
You can export it directly by:

```js
export { default as AXACheckbox } from './components/a-checkbox';
```

https://stackoverflow.com/questions/35665759/es6-how-can-you-export-an-imported-module-in-a-single-line
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Module_Redirects

Reply:
nice thanks

Reply:
will do asap